src/modules/cell.js

```diff
@@ -192,15 +192,18 @@
       );
     }
 
-    text = Array.isArray(text) ? text : [text];
+    const maxWidthIsDefind =  options.maxWidth !== undefined && options.maxWidth !== null 
+    text = Array.isArray(text) ? text : maxWidthIsDefind ? this.splitTextToSize(text, options.maxWidth) : [text];
```

Comment:
This behaves now slightly different than in the `text` method if an array is passed. We could discuss what's best here, but for the moment it should behave the same. See [jspdf.js#L3527](https://github.com/MrRio/jsPDF/blob/master/src/jspdf.js#L3527).

Reply:
@HackbrettXXX I just pushed a new commit which should make it behaveexactly the same as in the `text` method